pom.xml

```diff
@@ -311,6 +324,23 @@
                 <version>${slf4j.version}</version>
                 <scope>runtime</scope>
             </dependency>
+            <dependency>
+                <groupId>org.eclipse.rdf4j</groupId>
```

Comment:
rdf4j dependencies have to be removed (and are not sync anymore with core since a recent PR)

Reply:
It is to fix the error when we get a case with xiidm format :
curl -X GET "http://localhost:5000/v1/cases/c23b830e-92aa-4804-b911-614dc541349b?xiidm=true" -H "accept: */*"
com.powsybl.commons.PowsyblException: No implementation available for triple store rdf4j